src/api/java/com/minecolonies/api/util/constant/ColonyConstants.java

```diff
@@ -96,9 +96,10 @@
     public static final ResourceLocation BARBARIAN                    = new ResourceLocation(Constants.MOD_ID, "Barbarian");
     public static final ResourceLocation ARCHER                       = new ResourceLocation(Constants.MOD_ID, "ArcherBarbarian");
     public static final ResourceLocation CHIEF                        = new ResourceLocation(Constants.MOD_ID, "ChiefBarbarian");
-    public static final int              MAX_SIZE                     = Configurations.gameplay.maxBarbarianHordeSize;
-    public static final double           BARBARIANS_MULTIPLIER        = 0.5;
-    public static final double           ARCHER_BARBARIANS_MULTIPLIER = 0.25;
+    public static final double           BARBARIAN_SPAWN_SIZE         = ((double) Configurations.gameplay.spawnBarbarianSize * 0.1);
```

Comment:
Always complicated to calc statics like that, if the player changes the config from the ingame editor it wouldn't reload this variable then.

Reply:
I know the code was like that before, but ti would be nice to use the Configurations at the places directly, they are constants already basically.

Reply:
Done